01_Data/src/layers/sequelize/model/Authorization/IdToken.ts

```diff
@@ -28,4 +29,7 @@ export class IdToken extends Model implements IdTokenType {
   declare type: IdTokenEnumType;
 
   declare customData?: CustomDataType;
+
+  @HasOne(() => Authorization)
+  declare authorization: Authorization;
```

Comment:
It is possible that an IdToken is a IdTokenInfo.groupId token. Linking it this way does not allow for this distinction. This will need to be changed.